### ISSUE TITLE
Replace with rewrite?

### ISSUE THREAD

**Issue by danomatika** [Akira-Hayasaka/ofxGLEditor#10]
I named my ofxGLEditor rewrite "ofxEditor" but I had forgotten there already is an existing [ofxEditor](https://github.com/darrenmothersele/ofxEditor). Maybe it makes sense for my new ofxEditor to replace ofxGLEditor as a new "ofxGLEditor"? What do you think?

I mainly started working on the new code to fix issues with the Fluxus editor so it can be a drop in replacement for the current ofxGLEditor.

**Comment by Akira-Hayasaka** [comment]
Hi,
I agree with you, ofxGLEditor should be replaced with your new "ofxGLEditor".

Do you want to replace it on my current ofxGLEditor repository? If so, I already added you as a collaborator, so now you can freely replace it without PR.

Or do you want to make new repo for ofxGLEditor on your github account? If so, I will mark my ofxGLEditor as depreciated then put link to your new repo on readme. (I will not delete my repo for the purpose of preservation).

How do you think?

**Comment by danomatika** [comment]
I think it's best to keep things in one place, so I merged in the rewrite to ofxGLEditor. If you want to transfer the repo to me later on, we can do that, but for now we have the new code merged with the old code and a complete history. :D

**Comment by danomatika** [comment]
Also, there is a "original" tag for the last commit for the old version.

**Comment by Akira-Hayasaka** [comment]
OK, thank you!

**Comment by Akira-Hayasaka** [comment]
added livecoding example! 
using ofxLua

**Comment by danomatika** [comment]
Nice work. I updated the example and renamed it to "livecodingExample". I also added an isSelection() function so now we can just rerun the text in the current selection instead of the entire script if we want.

I'm not sure you're aware of it, but I've been working on replacing Luabind in ofxLua with bindings generated by swig. This means no BOOST and only a simple .cpp file that needs to be compiled, so ofxLua now works with the ProjectGenerator. It's currently in the `swig` branch: <https://github.com/danomatika/ofxLua/tree/swig>

Also, I've been keeping the actual project files for the examples out of the repository now since they can be generated using the ProjectGenerator, so I removed the xcodeproj, Makefile, etc. You only need to regenerate them on your end using the PG, the info on who to do that is in the Readme. It's easy. :D
